docs/hf_hub.rst

```diff
@@ -111,3 +111,42 @@ returns the model output for a given input. It can be used as::
 
 In the above code snippet, ``res`` will be a :class:`numpy.ndarray` containing
 the model's output.
+
+.. _hf_hub_gradio:
+
+Easily build user interfaces to your scikit-learn models
+--------------------------------------------------------
+`gradio <https://gradio.app/>`__ is a python library that lets you create interfaces on your model.
+It has a class called :class:`Interface` that lets you create application
```

Comment:
The link to `Interface`, as well as below `launch`, is not properly generated. This is because we lack the intersphinx mapping to the gradio docs.

To add those, we need to add an entry [here](https://github.com/skops-dev/skops/blob/cfea7cb5a741a86365eaa9bd4d56a86cc46cbea5/docs/conf.py#L93) which should look similar to this:

```python
    "gradio": ("https://www.gradio.app/docs/", None),
```

However, when I tried, I got this error:

`intersphinx inventory 'https://www.gradio.app/docs/objects.inv' not fetchable due to <class 'requests.exceptions.HTTPError'>: 404 Client Error: Not Found for url: https://www.gradio.app/docs/objects.inv`

@freddyaboulton can you help?

Reply:
Looking into it @BenjaminBossan ! I think the problem is that we don't generate the file expected by sphinx. Can you link to the gradio website in a way that doesn't rely on intersphinx in the meantime?

Reply:
I added a TODO comment in file in the meantime https://github.com/skops-dev/skops/pull/134/commits/b2a96a68b9a07ba2bcdc11016725e1d3c9324764